core/capabilities/webapi/trigger/trigger.go

```diff
@@ -38,6 +43,7 @@ type webapiTrigger struct {
 	ch             chan<- capabilities.TriggerResponse
 	config         webapicap.TriggerConfig
 	rateLimiter    *common.RateLimiter
+	rawConfig      *values.Map
```

Comment:
what data goes in this `rawConfig`?

Reply:
It's the config that is in the triggers config, ie allowedSenders, allowedTopics, RateLimiter..  I wanted to make the metadata exchange "future-proof" so that changes to the config don't require rewriting the exchange logic.